plugin/kubernetes/external.go

```diff
@@ -12,8 +12,8 @@ import (
 )
 
 // External implements the ExternalFunc call from the external plugin.
-// It returns any services matching in the services' ExternalIPs.
-func (k *Kubernetes) External(state request.Request) ([]msg.Service, int) {
+// It returns any services matching in the services' ExternalIPs and if enabled, headless endpoints..
+func (k *Kubernetes) External(state request.Request, headless bool) ([]msg.Service, int) {
```

Comment:
I could be missing it, but I don't the see the `headless` bool parameter checked/used in this function.

Reply:
yep, it should be used in the if you've mentined before.